src/components/MarkDown/index.tsx

```diff
@@ -20,12 +19,10 @@ const Container = styled.div`
   padding: ${({ theme }) => theme.space['1']};
 `
 
-const headingRenderer = ({
-  node,
+const MdHeading = ({
```

Comment:
Possible to use full word instead of acronyms ?

```suggestion
const MarkDownHeading = ({
```